corehq/apps/case_importer/do_import.py

```diff
@@ -606,7 +606,7 @@ def get_user(name):
                 return None
 
         def get_group(name):
-            return Group.by_name(self.domain, name, one=True)
+            return Group.by_name(self.domain, name)
```

Comment:
This argument was originally added by @dannyroberts in https://github.com/dimagi/commcare-hq/commit/129f0276cdefb2054368f8383fea2cbaf2fd7cc6#diff-af96256b43c9e9882610f6f5dde86a82d55e6f71430fd1b57183222a23b3b08aR106.
There were two instances that had `one=False` back then and now they both don't pass that overwrite anymore. I could not find anything on searching `one=False` in code that are could be usages of this method.